base_tier_validation/models/tier_validation.py

```diff
@@ -171,6 +181,17 @@ def _compute_validated_rejected(self):
             rec.rejected_message = rec._get_rejected_message()
             rec.to_validate_message = rec._get_to_validate_message()
 
+    def _compute_validation_status(self):
```

Comment:
Instead of adding a new compute, can you rely on _compute_validated_rejected to not have duplicated logic or a possible deviation in the way of calculating what "validated" means.

validated True and rejected False -> status validated
validated False and rejected True -> status rejected
validated False and rejected False + review_ids exist -> status pending

Reply:
Another option is to rely on this field, and let the other fields as computed for backwards compatibility.

Reply:
But the main driver continues to be the booleans, The transition can be done with backward compatibility in v17, but in current stable verions I won't change how the booleans values are computed to avoid surprises.

Reply:
OK, it's like that right now.

Reply:
No, it is not like that now. I prefer to do this https://github.com/OCA/server-ux/pull/730#discussion_r1363897379. Now the status is based on one logic and not on the booleans.

Reply:
Changes done.